### DISCUSSION TITLE
Enabling Embedded Web Browser on Netbeans v20

### DISCUSSION THREAD

**Discussion by hunterz-killer** [apache/netbeans#7119] [Q&A] [answered]
Hi,

How to Enable Embedded Web Browser on Netbeans v20

**Comment by matthiasblaesing** [comment, marked as answer]
1. Go to Tools -> Plugins -> Available Plugins
2. Ensure the list is current by activating "Check for Newest"
3. Look for "JavaFX Implementation for <OS> (<ARCH>)"
4. Install it
5. Restart

If it is not present, check the Installed tab and if not activated, activate the module. Again make sure you restart. The embedded browser is "Embedded Browser UI" and depends on OpenJFX.

**Reply by hunterz-killer** [reply]
Thanks. It works
